tests/testthat/_snaps/snapshot.md

```diff
@@ -22,11 +22,44 @@
       [1] "1"
     Message <simpleMessage>
       2
-    Warning <warning>
+    Warning <simpleWarning>
       3
-    Error <rlang_error>
+    Error <simpleError>
       4
 
+# empty lines are preserved
+
+    Code
+      f()
+    Output
+      1
+      
+    Message <simpleMessage>
+      2
+      
+    Warning <simpleWarning>
+      3
+      
+    Error <simpleError>
+      4
+      
+
+---
+
+    Code
+      f()
+    Output
+      1
+      
+    Message
+      2
+      
+    Condition
+      Warning in `f()`:
+      3
```

Comment:
@lionel- should this get a prefix? And I think something in rlang needs to change in order to preserve the trailing nl? (Alternatively, maybe we only care about this for output and messages?)

Reply:
I think something about how rlang/cli is reformatting this makes it impossible for testthat to recover the trailing nl here.